mmv1/third_party/terraform/services/compute/resource_compute_region_instance_group_manager_test.go.erb

```diff
@@ -367,6 +367,28 @@ func TestAccRegionInstanceGroupManager_stateful(t *testing.T) {
 	})
 }
 
+func TestAccRegionInstanceGroupManager_APISideListRecordering(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"name": fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10)),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckRegionInstanceGroupManagerDestroyProducer(t),
+		Steps: []resource.TestStep{
+			// {
+			// 	Config: testAccRegionInstanceGroupManager_statefulOrdered(context),
```

Comment:
Should these lines + config block be deleted, or uncommented? I _think_ just testAccRegionInstanceGroupManager_statefulUnordered covers the interesting parts of the change.

Reply:
I've deleted them, thanks for spotting it!